src/pubVerifToken.ts

```diff
@@ -122,6 +127,30 @@ export class TokenResponse implements TokenResponseProtocol {
     }
 }
 
+export class PublicVerifiableToken extends Token {
+    static async fetch(pt: PrivateToken): Promise<PublicVerifiableToken> {
+        const issuerUrl = await getIssuerUrl(pt.challenge.issuerName);
+        const client = new Client();
+        const tokReq = await client.createTokenRequest(pt);
+        const tokRes = await tokReq.send(issuerUrl, TokenResponse);
+        const token = await client.finalize(tokRes);
+        return token;
+    }
+
+    static deserialize(tokenTypeEntry: TokenTypeEntry, bytes: Uint8Array): PublicVerifiableToken {
```

Comment:
note: down the line, I think we should infer tokenTypeEntry from the bytes. Caller might not know the token type before deserialization.

Reply:
this will be handled in other PR when integrating VOPRF tokens.